common/client-libs/validator-client/src/validator_api/routes.rs

```diff
@@ -11,6 +11,7 @@ pub const ACTIVE: &str = "active";
 pub const REWARDED: &str = "rewarded";
 
 pub const COCONUT_BLIND_SIGN: &str = "blind-sign";
+pub const COCONUT_SIGNATURE: &str = "signature";
```

Comment:
It may also be worth adding an extra route level specifically for the bandwidth so:

```
/api/v1/coconut/bandwidth/blind-sign
/api/v1/coconut/bandwidth/partial-credential
```

Because the number of private and public parameters is pretty specific to bandwidth and isn't really useful in other cases, unless by chance, they use the same parameters and proofs. Bitter experience from the PCC.

Reply:
Extra route levels sounds good to me. I presume the `/api/` part is in reference to what we use on our own validator-api machine, as there is no reference to it in the code afaik. It's just our way of redirecting to local 8080 port.